content/docs/concepts/security/index.md

```diff
@@ -10,98 +10,160 @@ aliases:
+In both cases the authentication policies are stored in the Istio config store via a custom Kubernetes API,
+and are kept up to date (along with keys where appropriate) for each proxy by Pilot.
+Istio also lets you enable authentication in permissive mode to understand how a policy change would affect your security posture
+before it becomes effective.
 
-    * For example, let's say we have a workload pulling data from a multi-tenant database. If Alice ran this workload, she will be able to pull
-    a different set of data than if Bob ran this workload.
+For more details,  please read [authentication policy](docs/concepts/security/#authentication-policy).
 
-* Service accounts enable strong security policies by offering the flexibility to identify a machine, a user, a workload, or a group of workloads (different
-workloads can run as the same service account).
+### Authorization & Audit
 
-* The service account a workload runs as won't change during the lifetime of the workload.
+Istio provides fine-grained access control to enable application-level control and auditing on who accesses your service, API, or resource,
+using a variety of access control mechanisms.
+These include attribute and role-based access control as well as authorization hooks.
 
-* Service account uniqueness can be ensured with domain name constraint
+Find out more about Istio authorization in the [authorization guide](docs/concepts/security/#authorization)
 
-### Communication security
+The guide for auditing is coming soon!
 
-Service-to-service communication is tunneled through the client side [Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy. End-to-end communication is secured by:
+### Security configuration
 
-* Local TCP connections between the service and Envoy
+Istio makes it simple to configure and enable security features using Istio’s rich but easy-to-use policy creation
+and centralized policy management.
 
-* Mutual TLS connections between proxies
+Find out more about configuring security policies in the Authentication and Authorization guides in the following sections.
+
+### Using other security mechanisms
 
-* Secure Naming: during the handshake process, the client side Envoy checks that the service account provided by the server side certificate is allowed to run the target service
+While we strongly recommend using Istio security,
+Istio is flexible enough to allow you to plug in your own authentication and authorization mechanisms via our Mixer component.
+You can find out how to use and configure [Mixer plugins](/docs/concepts/policies-and-telemetry/#adapters) in Policies and Telemetry.
 
-### Key management
+### Upcoming features
 
-Istio supports services running on both Kubernetes pods and VM/bare-metal machines. We use different key provisioning mechanisms for each scenario.
+* Identity/CA pluggability to support bring-your-own-CA and bring-your-own-identity. I.e. Vault integration, Active Directory integration
```

Comment:
There is a spelling error here or a missing spelling addition to the TLD .spelling file to get the gating moving.

Reply:
Done.